src/NServiceBus.Transport.SQS.AcceptanceTests/Sending_in_compatibility_mode.cs

```diff
@@ -25,46 +25,37 @@ public class Context : ScenarioContext
 
         public class Sender : EndpointConfigurationBuilder
         {
-            public Sender()
-            {
+            public Sender() =>
                 EndpointSetup<DefaultServer>(builder =>
                 {
                     builder.ConfigureRouting().RouteToEndpoint(typeof(Message), typeof(Receiver));
-                    builder.ConfigureSqsTransport();
+                    builder.ConfigureSqsTransport().EnableV1CompatibilityMode = true;
```

Comment:
This is weird because apparently while migrating we have essentially removed this flag

https://github.com/Particular/NServiceBus.AmazonSQS/blob/release-5.6/src/NServiceBus.Transport.SQS.AcceptanceTests/Sending_in_compatibility_mode.cs#L33

Reply:
That stood out me as well. Is this test working as expected?

Reply:
I improved the test and added another one to make this better